entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+npx prisma db push
```

Comment:
What is the use of that ?

Reply:
`npx prisma db push` Is used to push the Prisma schema changes to the database without generating a migration.
Make the directly on db.